src/uu/nl/src/nl.rs

```diff
@@ -67,78 +68,108 @@ enum NumberFormat {
     RightZero,
 }
 
+pub mod options {
+    pub const FILE: &str = "file";
+    pub const BODY_NUMBERING: &str = "body-numbering";
+    pub const SECTION_DELIMITER: &str = "section-delimiter";
+    pub const FOOTER_NUMBERING: &str = "footer-numbering";
+    pub const HEADER_NUMBERING: &str = "header-numbering";
+    pub const LINE_INCREMENT: &str = "line-increment";
+    pub const JOIN_BLANK_LINES: &str = "join-blank-lines";
+    pub const NUMBER_FORMAT: &str = "number-format";
+    pub const NO_RENUMBER: &str = "no-renumber";
+    pub const NUMER_SEPARATOR: &str = "number-separator";
+    pub const STARTING_LINE_NUMER: &str = "starting-line-number";
+    pub const NUMBER_WIDTH: &str = "number-width";
+}
+
 pub fn uumain(args: impl uucore::Args) -> i32 {
     let args = args.collect_str();
 
-    let mut opts = getopts::Options::new();
+    let matches = App::new(executable!())
+        .name(NAME)
+        .version(VERSION)
+        .usage(USAGE)
+        .arg(Arg::with_name(options::FILE).hidden(true).multiple(true))
+        .arg(
+            Arg::with_name(options::BODY_NUMBERING)
+            .short("b")
+            .long(options::BODY_NUMBERING)
+            .help("use STYLE for numbering body lines")
+            .value_name("SYNTAX")
+        )
+        .arg(
+            Arg::with_name(options::SECTION_DELIMITER)
+            .short("d")
+            .long(options::SECTION_DELIMITER)
+            .help("use CC for separating logical pages")
+            .value_name("CC")
+        )
+        .arg(
+            Arg::with_name(options::FOOTER_NUMBERING)
+            .short("f")
+            .long(options::FOOTER_NUMBERING)
+            .help("use STYLE for numbering footer lines")
+            .value_name("STYLE")
+        )
+        .arg(
+            Arg::with_name(options::HEADER_NUMBERING)
+            .short("h")
+            .long(options::HEADER_NUMBERING)
+            .help("use STYLE for numbering header lines")
+            .value_name("STYLE")
+        )
+        .arg(
+            Arg::with_name(options::LINE_INCREMENT)
+            .short("i")
+            .long(options::LINE_INCREMENT)
+            .help("line number increment at each line")
+            .value_name("")
```

Comment:
I see that the existing implementation has empty value but GNU implementation has `NUMBER` for value name could you also add it?

```rust
       -i, --line-increment=NUMBER
              line number increment at each line
```